.github/workflows/release.yml

```diff
@@ -32,12 +41,35 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-      - name: Publish to PyPI
-        if: steps.changesets.outputs.published == 'true'
+      - name: Build Python package
         run: |
-          pip install twine build
-          cd packages/mystmd-py
-          bash scripts/pypi-deploy.sh
-        env:
-          TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
-          TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
+          pip install build
+          # Copy over needed files
+          cp ../mystmd/dist/myst.cjs src/mystmd_py/
+          cp ../mystmd/package.json .          
+          python -m build
+        working-directory: packages/mystmd-py
+      - name: Store the distribution packages
+        uses: actions/upload-artifact@v4
+        with:
+          name: python-package-distributions
+          path: packages/mystmd-py/dist/
+  release-py:  
+    name: Release Python
+    runs-on: ubuntu-latest
+    if: ${{ needs.release-npm.outputs.published || inputs.forcePublishPyPI }}
```

Comment:
Should this be `steps.changesets.outputs.published == 'true'` or does that not work across jobs?